frontend/src/scenes/surveys/SurveyAppearance.tsx

```diff
@@ -122,6 +128,24 @@ export function SurveyAppearance({
                             />
                         </>
                     )}
+                    <div className="mt-2">
+                        <LemonCheckbox
+                            label={
+                                <div className="flex items-center">
+                                    <span>Hide PostHog branding</span>
+                                    {!whitelabelAvailable ? (
+                                        <Tooltip title="Upgrade to PostHog Scale to hide PostHog branding">
+                                            <IconLock className="ml-2" />
+                                        </Tooltip>
+                                    ) : null}
+                                </div>
+                            }
+                            onChange={(checked) => onAppearanceChange({ ...appearance, whiteLabel: checked })}
+                            disabledReason={
+                                !whitelabelAvailable ? 'Upgrade to PostHog Scale to hide PostHog branding' : null
```

Comment:
```suggestion
                                !whitelabelAvailable ? 'Upgrade to any paid plan to hide PostHog branding' : null
```

Reply:
Makes sense! This was copy pasta'd from the "sharing modal" component, so I can update it there too